components/ws-manager-api/go/BUILD.yaml

```diff
@@ -10,4 +10,4 @@ packages:
       - "go.sum"
     config:
       packaging: library
-      dontTest: false
+      dontTest: true
```

Comment:
Non-blocking, question

@Furisto why disable? Just curious.

Reply:
Ah was not aware of that but it makes sense. Currently the test we have (for the webhook) fails. Webhooks will be removed with the next PR.

Reply:
> Webhooks will be removed with the next PR which means the test will be removed as well.

Do we have a written reason why is this being done? What's the replacement?

Reply:
This was a suggestion from @csweichel. Currently there is just nothing that we would want to validate.

Reply:
Let's talk about that in the next sync. One of the goals of the webhook is to avoid persistence with invalid values and wasting sync loops in the controller.

Reply:
@Furisto was the thought to use CEL, instead of webhooks to do validation? [This article](https://blog.rewanthtammana.com/kubernetes-crd-validation-with-cel-and-kubebuilder-marker-comments) suggests we cannot use CEL until Kubernetes 1.25 (we're currently on 1.23, and cannot move to 1.24 yet).